halide.cmake

```diff
@@ -15,18 +15,22 @@ cmake_minimum_required(VERSION 3.3)
 #
 
 # Add the include paths and link dependencies for halide_image_io.
-function(halide_use_image_io TARGET)
-  foreach(PKG PNG JPEG)
-    find_package(${PKG} QUIET)
-    if(${PKG}_FOUND)
-      target_compile_definitions(${TARGET} PRIVATE ${${PKG}_DEFINITIONS})
-      target_include_directories(${TARGET} PRIVATE ${${PKG}_INCLUDE_DIRS})
-      target_link_libraries(${TARGET} PRIVATE ${${PKG}_LIBRARIES})
+add_library(halide_image_io INTERFACE)
```

Comment:
No, that should be bumped.

Reply:
Nit: Do we have any meaningful concern about people being unwilling to upgrade CMake? (LLVM only requires 3.4.3, amazingly...)

Reply:
No, I don't think we do. The latest version can be installed locally with pip, no need for a system-wide install. VS2019 comes bundled with 3.15. Older VS users can install locally and generate vcxproj files instead of using built-in support, as they always have.

Reply:
I'm not aware of any backwards compatibility issues with CMake. To their credit, cmake is pretty much infinitely backwards compatible.